addons/mrp_subcontracting/i18n/mrp_subcontracting.pot

```diff
@@ -366,7 +366,7 @@ msgstr ""
 #. odoo-python
 #: code:addons/mrp_subcontracting/models/stock_warehouse.py:0
 #: code:addons/mrp_subcontracting/models/stock_warehouse.py:0
-#: model:stock.route,name:mrp_subcontracting.route_resupply_subcontractor_mto
+#: code:addons/mrp_subcontracting/wizard/product_replenish.py:0
```

Comment:
```suggestion
#: code:addons/mrp_subcontracting/wizard/product_replenish.py:0
#: model:stock.route,name:mrp_subcontracting.route_resupply_subcontractor_mto
```
If you remove that line, then you'll break the translation the record itself defined here:
https://github.com/odoo/odoo/blob/55d8434a8ca12d83c89e0301ce31dc54f6395d98/addons/mrp_subcontracting/data/mrp_subcontracting_data.xml#L4-L8